distribution/ddf-common/src/main/resources-filtered/etc/system.properties

```diff
@@ -216,6 +216,12 @@ org.codice.ddf.http.access.log.enabled=false
 #
 org.codice.ddf.security.ecp.enabled=true
 
+#
+# Security Logging
+#
+security.logger.extra_attributes=CountryOfAffiliation
```

Comment:
Is this an appropriate default for DDF?

Reply:
Good question. While I don't think it's a bad idea to include this block in the file, I think we should probably leave this attribute commented out. Downstream projects that have need of it can add if they choose.

Reply:
That's a good point. I've gone ahead and removed it.